### PR TITLE
Feat:  theming docs

### PR DESCRIPTION
**Pull request by eduardoformiga**
## What's the purpose of this pull request?

Adds the theming section to nextra documentation.

- [x] Brandless
- [x] Customizing
- [x] Themes: midnight and soft-blue.

The `WebFont` step will be implemented in [this task](https://vtex-dev.atlassian.net/browse/FS-854). 

## How to test it?

Check the documentation in the preview link.

### CODE REVIEW

File: apps/site/pages/docs/brandless.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+**Structural Styles**, the first layer, has foundational styles that lay things out in common design patterns for different use cases. It also covers a component's fundamental interactions, like showing/hiding a dropdown or moving a toggle’s knob to make it active.
+
+The second, **Theme**, is where the branding lives and what truly differentiates one UI from another. Graphic elements—like typography (and type scale), colors, imagery, borders, spacing, & more—are controlled by design tokens that a developer can easily tweak to create a unique look and feel.
+
+<img
+  src="https://vtexhelp.vtexassets.com/assets/docs/src/theming-intro___1d021c8548ba2338faba6dd2995bf7ce.png"
+  align="right"
+  style={{ width: '70%', display: 'block', margin: '0 auto' }}
+/>
```
Comment (hellofanny) [inline apps/site/pages/docs/brandless.mdx:30]:
```suggestion
<div style={{ display: 'flex', marginTop: '12px' }}>
<img
  src="https://vtexhelp.vtexassets.com/assets/docs/src/theming-intro___1d021c8548ba2338faba6dd2995bf7ce.png"
  style={{ width: '95%', display: 'block', margin: '0 auto' }}
/>
</div>
```

Comment (hellofanny) [reply apps/site/pages/docs/brandless.mdx:30]: I think it looks better in a single row. WTDY?

image

File: apps/site/pages/docs/customizing.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+**Design tokens** are parameters that affect all UI look 'n' feel and they are the **main configuration** file of a Theme. Component-specific tokens often inherit from global tokens but are named specifically to apply those tokens in component development.
+
+</header>
+
+## Global Tokens
+
+FastStore uses a default theme called `Brandless`, which provides structural styles for your components and pages. Please read [Brandless's page](/docs/brandless) to further information about our themeable architecture.
+
+Its presets are defined on `src/styles/global/tokens.scss` and it's separated in these main areas:
```
Comment (hellofanny) [inline apps/site/pages/docs/customizing.mdx:15]:
```suggestion
Its presets are defined on `src/base/tokens.scss` and it's separated in these main areas:
```

Comment (hellofanny) [reply apps/site/pages/docs/customizing.mdx:15]: We are referring to ui package right?

`ui/src/base/tokens.scss`

File: apps/site/pages/docs/customizing.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      </>
+    }
+  ></SectionItem>
+</SectionList>
+
+---
+
+## Build your new theme
+
+1. Create a new `.scss` file on `src/themes` and overwrite the tokens you want: e.g. `my-theme.scss`
```
Comment (hellofanny) [inline apps/site/pages/docs/customizing.mdx:80]: I think we need to be more specific here. 
Where should we add this src/themes folder? In the store project?

**Review by renatamottam — APPROVED** [review]
Nice work! Left a suggestion below :)

File: apps/site/pages/docs/customizing.mdx (hunk 2 above)
Comment (eduardoformiga) [reply apps/site/pages/docs/customizing.mdx:15]: Nice catch!
Well, I think this is not the best way to refer to the base tokens yet. 
What do you think about adding the `tokens.scss` content at the end of the brandless section? @hellofanny @renatamottam

Comment (eduardoformiga) [reply apps/site/pages/docs/customizing.mdx]: WDYT about [this changes](https://github.com/vtex/faststore/pull/1636/commits/98d6f5e34f8862ff4489b0e25454a54839ddcce2)?

File: apps/site/pages/custom-nextra.css [hunk 4]
```diff
@@ -64,3 +64,49 @@ h4.nx-tracking-tight {
 [data-fs-payment-methods-title] h3 {
   margin-top: auto;
 }
+
```
[the remark this answers is not in the document]
Comment (eduardoformiga) [reply apps/site/pages/custom-nextra.css:67]: I didn't find an easy way to override the default content provided by nextra. 
And I think we shouldn't change the values from nextra pre-defined (tailwind?) classes.
Luckily, nextra provides a `full` [layout mode](https://nextra.site/docs/docs-theme/page-configuration#layouts) we can use on specific pages, and we only miss the right sidebar. I [changed it here](https://github.com/vtex/faststore/pull/1636/commits/47248b692fb48ad22fee0427d196afc85c2d0cdd). You can check in the preview link. What do you think? @renatamottam @hellofanny 
|Before|After|
|-|-|
|Screen Shot 2023-02-14 at 19 19 13|Screen Shot 2023-02-14 at 19 19 22|
|Screen Shot 2023-02-14 at 19 19 38|Screen Shot 2023-02-14 at 19 19 52|

File: apps/site/pages/docs/customizing.mdx (hunk 2 above)
Comment (hellofanny) [reply apps/site/pages/docs/customizing.mdx]: Agreed!
I like this latest changes :) It's better to show the file path this way 👍

File: apps/site/pages/docs/customizing.mdx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+      </>
+    }
+  ></SectionItem>
+</SectionList>
+
+---
+
+## Build your new theme
+
+1. Create a new `.scss` file on `src/themes` (e.g. `my-theme.scss`) in your starter project, and add the tokens you want to overwrite inside the `.theme` class:
```
Comment (hellofanny) [inline apps/site/pages/docs/customizing.mdx:80]:
```suggestion
1. Create a new `.scss` file on `src/themes` (e.g. `my-theme.scss`) in your [starter project](https://github.com/vtex-sites/starter.store), and add the tokens you want to overwrite inside the `.theme` class:
```

Comment (hellofanny) [reply apps/site/pages/docs/customizing.mdx:80]: Should we link the starter repo here? 🤔

File: apps/site/pages/custom-nextra.css (hunk 4 above)
[the remark this answers is not in the document]
Comment (hellofanny) [reply apps/site/pages/custom-nextra.css:67]: heyyy! I think it works! Since there aren't many sections on this page, I'm fine with the full layout mode.

**Review by hellofanny — APPROVED** [review]
Great job formssss! 🚀 🚀  Congrats!

**Review by ArthurTriis1 — APPROVED** [review]

**Pull request merged**